tests/dummy/app/app.js

```diff
@@ -6,6 +6,7 @@ import config from './config/environment';
 let App;
 
 Ember.MODEL_FACTORY_INJECTIONS = true;
+Ember.run.backburner.DEBUG = true;
```

Comment:
Do we intend to keep this flag on? I'm curious how your experience fits with the warning in the docs:

From https://guides.emberjs.com/v2.4.0/configuring-ember/debugging/#toc_errors-within-code-ember-run-later-code-a-href-https-github-com-ebryn-backburner-js-backburner-js-a
> Backburner has support for stitching the stacktraces together so that you can track down where an erroring Ember.run.later is being initiated from. Unfortunately, this is quite slow and is not appropriate for production or even normal development.